documentation/staging/content/quickstart/get-images.md

```diff
@@ -23,7 +23,11 @@ weight: 3
 
     a. First time users, follow these [directions]({{< relref "/userguide/base-images/ocr-images#obtain-images-from-the-oracle-container-registry" >}}).
 
-    b. Find and then pull the WebLogic 12.2.1.4 install image:
+    b. Find and then pull the WebLogic 12.2.1.4 General Availability (GA) installation image.
+
+   {{% notice warning %}}
+   GA images are suitable for demonstration and development purposes _only_ where the environments are not available from the public Internet; they are **not acceptable for production use**. In production, you should always use CPU (patched) images from [OCR]({{< relref "/userguide/base-images/ocr-images.md" >}}) or create your images using the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/userguide/tools/create-image/) (WIT) with the `--recommendedPatches` option. For more guidance, see [Apply the Latest Patches and Updates](https://www.oracle.com/pls/topic/lookup?ctx=en/middleware/standalone/weblogic-server/14.1.1.0&id=LOCKD-GUID-2DA84185-46BA-4D7A-80D2-9D577A4E8DE2) in _Securing a Production Environment for Oracle WebLogic Server_.
```

Comment:
I put in the link to the WIT doc because that is where the `--recommended patches` option is described. (Yes, we have one example using it in the Create a custom base image section.) I can change it.

Reply:
> General comment: I could wrong, but IIRC there are more references to GA images than are here -- for example in the Azure ".txt" files. I have not checked, but will do so if you'd like me too.

@tbarnes-us I did search the Azure ".txt" files and made changes there. If I missed any, please point them out. (PS Their use of .txt files instead of .md files makes the x-refs more difficult. Feel free to make any suggestions you can think of for improvements.)